Various changes to roles, some added.

JamesLaw222 opened the pull request:
# Description

I have:

- Added the Skinchanger, Elder, and Diviner pages, with notes and tips for each.
- Updated the Ghoul and Voodoo Doctor pages to reflect recent changes to how they work (item stealing for the former, new Zombie mechanics for the latter). I have added some full-stops in the Ghoul page.
- Updated the Strongman page to be more detailed and specific. 

Everything follows the specific information from #changelog or generally accepted information. I believe it all to be correct, but will not check the box in fear of being wrong.

# Checklist

I have checked the following _(mark as completed with [x])_:

- [x] The content is accurate (and confirmed with the mods)
- [x] Casing and wording of keywords such as role names, factions, etc, is all correct
- [x] The Markdown is valid
- [ ] It renders correctly _(follow README for instructions on previewing)_
- [x] The sidebar links are correct and in the right sections _(follow README for instructions on previewing)_

JamesLaw222 committed:
Added item-stealing change

JamesLaw222 committed:
Created Skinchanger

JamesLaw222 committed:
Updated to include specific notes and tips

JamesLaw222 committed:
Created Elder

JamesLaw222 committed:
Rename Elder.mdx to elder.mdx

JamesLaw222 committed:
Rename Skinchanger.mdx to skinchanger.mdx

JamesLaw222 committed:
Updated to include new role changes

JamesLaw222 committed:
Created Diviner page

JamesLaw222 committed:
Updated Seer check for Undead games

JamesLaw222 committed:
Removed reference to Undead being new

JamesLaw222 committed:
Removed syntax issue that came from moving repos

JamesLaw222 committed:
Improved tips and fixed spelling error

Kirschstein reviewed (commented):
Looks good to me, will let Shane and Danny take a look over as well

StormPooper commented on src/guides/village/strongman.mdx:
```diff
@@ -6,11 +6,26 @@ route: /roles/Strongman
+## History
+
+<Timeline lineColor="white">
+  <Event interval="2018-12-xx">Introduced.</Event>
```
Does this work? I'd expect it to need an actual date.

JamesLaw222 replied:
I've updated it to match the first reference to the role being in a game on Discord. It may not be entirely accurate, but sans knowing precisely which game it was first introduced in, it is the best solution.

The date is now 2018-12-08. If it was actually introduced before this it can be changed once more, but it probably isn't really necessary to do so.

StormPooper commented on src/guides/neutral/skinchanger.mdx:
```diff
@@ -0,0 +1,41 @@
+---
+name: Skinchanger
```
#211

JamesLaw222 replied:
Updated to hopefully reach a good version that covers all of the ground we both wanted to cover, using pieces of both versions. Hans has expressed satisfaction with this.

StormPooper replied:
@TheHans255 you'll need to update/recreate the rest of your PR once this is merged in.

JamesLaw222 committed:
Updated introduction date to match first reference

I used the first reference to the Strongman as an existing role as the date for the role's introduction. I am unsure which game was the first.

JamesLaw222 committed:
Updated to include details found in Hans' version

In his PR, Hans has his own version of the Skinchanger page. This is an attempt to combine the information of both.
